src/elements/content-preview/PreviewNavigation.js

```diff
@@ -27,28 +38,45 @@ const PreviewNavigation = ({ collection = [], currentIndex, intl, onNavigateLeft
     }
 
     return (
-        <>
-            {hasLeftNavigation && (
-                <PlainButton
-                    className="bcpr-navigate-left"
-                    onClick={onNavigateLeft}
-                    title={intl.formatMessage(messages.previousFile)}
-                    type="button"
-                >
-                    <IconNavigateLeft />
-                </PlainButton>
-            )}
-            {hasRightNavigation && (
-                <PlainButton
-                    className="bcpr-navigate-right"
-                    onClick={onNavigateRight}
-                    title={intl.formatMessage(messages.nextFile)}
-                    type="button"
-                >
-                    <IconNavigateRight />
-                </PlainButton>
-            )}
-        </>
+        <NavRouter history={history}>
+            <Route path={['/:activeTab/:deeplink', '/']}>
```

Comment:
Are these paths comprehensive enough to ensure that these navigation buttons always render? Should `'/'` be `'*'`?

Reply:
Nvm, just read that `'/'` will always match the URL